spec/requests/foods_spec.rb

```diff
@@ -0,0 +1,98 @@
+# sticking to rails and rspec-rails APIs to keep things simple and stable.
+RSpec.describe '/foods', type: :request do
+  # This should return the minimal set of attributes required to create a valid
+  # Food. As you add validations to Food, be sure to
+  # adjust the attributes here as well.
+  let(:valid_attributes) do
+    skip('Add a hash of attributes valid for your model')
+  end
+  let(:invalid_attributes) do
+    skip('Add a hash of attributes invalid for your model')
+  end
+  describe 'GET /index' do
+    it 'renders a successful response' do
+      Food.create! valid_attributes
+      get foods_url
+      expect(response).to be_successful
+    end
+  end
+  describe 'GET /show' do
+    it 'renders a successful response' do
+      food = Food.create! valid_attributes
+      get food_url(food)
+      expect(response).to be_successful
+    end
+  end
+  describe 'GET /edit' do
+    it 'renders a successful response' do
+      food = Food.create! valid_attributes
+      get edit_food_url(food)
+      expect(response).to be_successful
+    end
+  end
+  describe 'POST /create' do
+    context 'with valid parameters' do
+      it 'redirects to the created food' do
+        post foods_url, params: { food: valid_attributes }
+        expect(response).to redirect_to(food_url(Food.last))
+      end
+    end
+    context 'with invalid parameters' do
+      it "renders a successful response (i.e. to display the 'new' template)" do
+        post foods_url, params: { food: invalid_attributes }
+        expect(response).to be_successful
+      end
+    end
+  end
+  describe 'PATCH /update' do
+    context 'with valid parameters' do
+      let(:new_attributes) do
+        skip('Add a hash of attributes valid for your model')
+      end
+      it 'updates the requested food' do
+        food = Food.create! valid_attributes
+        patch food_url(food), params: { food: new_attributes }
+        food.reload
+        skip('Add assertions for updated state')
+      end
+      it 'redirects to the food' do
+        food = Food.create! valid_attributes
+        patch food_url(food), params: { food: new_attributes }
+        food.reload
+        expect(response).to redirect_to(food_url(food))
+      end
+    end
+    context 'with invalid parameters' do
+      it "renders a successful response (i.e. to display the 'edit' template)" do
+        food = Food.create! valid_attributes
+        patch food_url(food), params: { food: invalid_attributes }
+        expect(response).to be_successful
+      end
+    end
+  end
+  describe 'DELETE /destroy' do
+    it 'destroys the requested food' do
+      food = Food.create! valid_attributes
+      expect do
+        delete food_url(food)
+      end.to
+      change(Food, :count).by(-1)
+    end
+    it 'redirects to the foods list' do
+      food = Food.create! valid_attributes
+      delete food_url(food)
+      expect(response).to redirect_to(foods_url)
+    end
+  end
+end
```

Comment:
- [ ] Well done adding tests. BUt only tests in the feature folder is passing. Request tests are pending when run the `rspec`. Kİnldy fix it. 
![image](https://user-images.githubusercontent.com/38635158/177006934-c18da4df-afd4-47aa-bd2f-5601f4597f7e.png)

![image](https://user-images.githubusercontent.com/38635158/177006902-28780bc4-481a-4cfa-9616-e24255c1097f.png)

Reply:
@ahmetbozaci, Thanks for reviewing. We have worked on this.